hoomd/BoxResizeUpdater.h

```diff
@@ -14,51 +14,84 @@
 
 #include "Updater.h"
 #include "Variant.h"
+#include "BoxDim.h"
 
 #include <memory>
+#include <string>
+#include <stdexcept>
 #include <pybind11/pybind11.h>
 
 #ifndef __BOXRESIZEUPDATER_H__
 #define __BOXRESIZEUPDATER_H__
 
-//! Updates the simulation box over time
-/*! This simple updater gets the box lengths from specified variants and sets those box sizes
-    over time. As an option, particles can be rescaled with the box lengths or left where they are.
-
-    \ingroup updaters
+/// Updates the simulation box over time
+/** This simple updater gets the box lengths from specified variants and sets
+ * those box sizes over time. As an option, particles can be rescaled with the
+ * box lengths or left where they are.
+ * \ingroup updaters
 */
 class PYBIND11_EXPORT BoxResizeUpdater : public Updater
     {
     public:
-        //! Constructor
+        /// Constructor
         BoxResizeUpdater(std::shared_ptr<SystemDefinition> sysdef,
-                         std::shared_ptr<Variant> Lx,
-                         std::shared_ptr<Variant> Ly,
-                         std::shared_ptr<Variant> Lz,
-                         std::shared_ptr<Variant> xy,
-                         std::shared_ptr<Variant> xz,
-                         std::shared_ptr<Variant> yz);
-
-        //! Destructor
+                         pybind11::object initial_box,
+                         pybind11::object final_box,
+                         std::shared_ptr<Variant> variant);
+
+        /// Destructor
         virtual ~BoxResizeUpdater();
 
-        //! Sets parameter flags
-        void setParams(bool scale_particles);
+        /// Sets particle scaling. When true, particle positions are scaled with the box.
+        void setScaleParticles(bool scale_particles)
+            {
+            m_scale_particles = scale_particles;
+            }
+
+        /// Gets particle scaling setting
+        bool getScaleParticles() {return m_scale_particles;}
```

Comment:
Is this the clang-formatting style that HOOMD is following now (all on one line for getters), or are you just waiting to run clang-format later to fix the formatting?

Reply:
`clang-format` formatting for HOOMD is not yet in place.